.buildconfig.yml

```diff
@@ -60,6 +60,10 @@ projects:
     path: components/feature/media
     description: 'Feature component for website media related features.'
     publish: true
+  feature-notifications:
```

Comment:
naming nit: let's maybe be more explicit and pick `feature-web-notifications` or `feature-webnotifications` since just "notifications" can mean a lot and could be confusing since there are other components showing notifications.

Reply:
Updated!